app/javascript/courses/students/types/CoursesStudents__Level.re

```diff
@@ -2,24 +2,36 @@ type t = {
   id: string,
   name: string,
   number: int,
-  teamsInLevel: int,
+  studentsInLevel: int,
```

Comment:
I think we could use both `teamsInLevel` and `studentsInLevel` values here.

1. Use the `teamsInLevel` value in the distribution bar itself. i.e., the numbers shown on the outside. For courses without teams, it would match student count. For courses with teams, it would then show teams count. No extra work.
2. If `teamsInLevel != studentsInLevel`, then the hover popup for levels can include **both** values. If they're equal, then it can just list students count - the way it behaves right now.